var/spack/repos/builtin/packages/anicalculator/package.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+
+from spack.package import *
+
+
+class Anicalculator(Package):
+    """This tool will calculate the bidirectional average nucleotide identity
+    (gANI) and Alignment Fraction (AF) between two genomes.
+
+    Note: A manual download is required for ANIcalculator.
+    Spack will search your current directory for the download file.
+    Alternatively, add this file to a mirror so that Spack can find it.
+    For instructions on how to set up a mirror, see
+    https://spack.readthedocs.io/en/latest/mirrors.html"""
+
+    homepage = "https://ani.jgi.doe.gov/html/download.php?"
+    manual_download = True
+
+    version("1", sha256="236596a9a204cbcad162fc66be3506b2530b1f48f4f84d9647ccec3ca7483a43")
+
+    depends_on("perl@5:")
```

Comment:
The default deptype is build/link. Since this package isn't compiled, I don't think it's being used. Should it be "run"?

Reply:
yes, that's probably correct. I didn't realize that was the default dependency type. I'll test it with that change, I previously just had it print the help message but didn't put any data through it.

Reply:
tested with real data and it works